### ISSUE TITLE
Issue observed on Pre-release branch

### ISSUE THREAD

**Issue by pavanpatil1** [rtCamp/rtMedia#1890]
```
Environment: 
WordPress: v5.9.3
buddypress: v10.2.0
rtMedia: Pre-release branch 
```

- [x] PHP notice is displayed after activating the plugin. 

Below PHP notice is displayed on debug log after activating the plugin. 

`PHP Notice:  Trying to access array offset on value of type bool in  \app\public\wp-content\plugins\rtMedia\app\admin\RTMediaAdmin.php on line 110`

https://user-images.githubusercontent.com/44057535/163426556-005d1768-bafc-4752-a59a-0e07bdc4318b.mp4

---

- [ ] Count for the album is visible but the album is not visible on the Groups albums page. 

On the Groups albums page, the count for the default album is visible but the album is not visible. 

**Steps to reproduce:** 
1. Install and activate the rtMedia plugin. Create a group and enable media in the group from rtMedia -> BuddyPress. 
2. Navigate to the group media page and click on albums. 
3. Observe the issue. 

Expected: Count should not be visible if the album is not present. 

https://user-images.githubusercontent.com/44057535/163515824-023e9cf5-92e8-4fec-b74f-69b1b372f35a.mp4

---

- [x] Console error for Masonry view is displayed on the member's activity page. 

**Steps to reproduce:** 
1. Enable the Masonry view from rtMedia -> Display settings. 
2. Navigate to the Activity/Members activity page and observe the console error. (Add a comment on any media to view the console error). 

Below console error is displayed. 
`masonry not initialized. Cannot call methods, i.e. $().masonry("reload")`

![image](https://user-images.githubusercontent.com/44057535/163516578-7d54c3b1-b2e8-45e1-b9df-0f98a3ffe719.png)

---

- [x] DB error displayed while editing the album. 

While editing the album, the database error is displayed in debug log while moving the media to another album. 

`[WordPress database error You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'WHERE `id` = '5'' at line 1 for query UPDATE `wp_rt_rtm_media` SET  WHERE `id` = '5' made by require('wp-blog-header.php'), require_once('wp-includes/template-loader.php'), apply_filters('template_include'), WP_Hook->apply_filters, RTMediaRouter->template_include, RTMediaTemplate->set_template, RTMediaTemplate->check_return_edit, RTMediaTemplate->save_edit, RTMediaTemplate->save_album_edit, RTMediaMedia->update, RTDBModel->update`

The above error is displayed in the log. 

https://user-images.githubusercontent.com/44057535/163523633-89819749-971e-4480-8627-b8ef7aed56d6.mp4

**Comment by Mukulsingh27** [comment]
Issue number 3 :-
`Console error for Masonry view is displayed on the member's activity page.`
Is not reproducible

**Comment by pavanpatil1** [comment]
Fixed in #1891. Hence closing this issue
